src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -141,6 +141,11 @@ class DefaultSource
         new EventsRelation(config)(sqlContext)
       case "files" =>
         new FilesRelation(config)(sqlContext)
+      case "filecontent" =>
+        val inferSchema = toBoolean(parameters, "inferSchema")
```

Comment:
I guess we can add inferSchemaLimit parameter support later if needed (looking at RawTableRelation having it)

Reply:
Yes, I'm still not totally certain about how it works under the hood, but I *think* as it is it isn't reading it twice, so it might not lead to any improvement.